### ISSUE TITLE
Pass the element that triggered the handler

### ISSUE THREAD

**Issue by mhulse** [kylefox/jquery-modal#237]
I may be missing it, but I can't seem to figure out how to use JS to get at the `rel="modal:open"` element that triggered the modal.

 Docs say:

```js
modal.$elm;       // Original jQuery object upon which modal() was invoked.
modal.options;    // Options passed to the modal.
modal.$blocker;   // The overlay element.
```

But, why not pass the callee element, too?

Looking like I will have to not use `rel="modal:open"` and setup my own event handling. I don't mind doing this ... Just seems like something that should be passed into, or somehow accessible from, the custom events params.

Or, am I missing something? 🤷‍♂️ 

Anyway, thanks for the code!

**Comment by kylefox** [comment]
This is now available in 0.9.2 https://github.com/kylefox/jquery-modal/pull/238

**Comment by mhulse** [comment]
Awesome! Thanks!!!
